src/mmap_unix.rs

```diff
@@ -210,15 +217,16 @@ impl MmapRegion {
 }
 
 impl AsSlice for MmapRegion {
-    // Returns the region as a slice
-    // used to do crap
+    /// Returns the region as a slice.
     unsafe fn as_slice(&self) -> &[u8] {
         // This is safe because we mapped the area at addr ourselves, so this slice will not
         // overflow. However, it is possible to alias.
         std::slice::from_raw_parts(self.addr, self.size)
     }
 
-    // safe because it's expected interior mutability
+    /// Returns the region as a mutable slice.
+    ///
+    /// It's safe because interior mutability is expected.
```

Comment:
Unsafe functions typically have a `# Safety` section. You can check an example here: https://rust-lang.github.io/api-guidelines/documentation.html#function-docs-include-error-panic-and-safety-considerations-c-failure

Reply:
Fixed.